ntoskrnl/io/iomgr/deviface.c

```diff
@@ -7,6 +7,7 @@
  * PROGRAMMERS:     Filip Navara (xnavara@volny.cz)
  *                  Matthew Brace (ismarc@austin.rr.com)
  *                  Hervé Poussineau (hpoussin@reactos.org)
+ *                  Oleg Dubinskiy (oleg.dubinskij30@gmail.com)
```

Comment:
@oleg-dubinskiy Do you have an "official" `@reactos.org` mail? If so, I would suggest you use it.

Reply:
Yeah, I have it starting from recently. Somewhere when I officialy became a developer. I remember Mark made it for me. But my main email I'm using is Gmail, and I did not use ReactOS mail yet, so if it is necessarily (is it?), I will change it to my ReactOS mail, and I probably should use it in all my previous and new copyrights I think.